test/sasdataloader/test/utest_abs_reader.py

```diff
@@ -95,6 +100,7 @@ def test_igor_checkdata(self):
             Data1D defaults changed. Otherwise the
             tests won't pass
         """
+        self.data = Loader().load("MAR07232_rest.ASC")
```

Comment:
Ok, so `self.data` should still be set in `igor_reader.setup`. However, this attribute should be created by an `IgorReader` object rather than the generic `Loader` object. The reason being that an `IgorReader` should definitely be able to read a file of this type.
As it stands the `Loader` object delegates to `IgorReader`, which fails with recent numpy, and resorts to using a different type of reader, which also fails. The error messages one sees is that some other reader failed to read this file, rather than the ultimate cause of `IgorReader` failing to read the file.